DuckDuckGo/Icon.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Icon.swift
```

Comment:
I wonder if it would be better to call this and the manager `AppIcon` - I guess we might have other icon stuff in the app and this will make it less ambiguous, what do you think?

Reply:
Good point! I agree 👍